### PR TITLE
Used fake backend to fetch data and show on the UI

### PR DESCRIPTION
**Pull request by Mahi24-prog**
### Added API calls to fetch data from fake backend and showed it on the UI

https://user-images.githubusercontent.com/59019169/161243786-3c94b06b-1af0-488e-8353-450e3391c7e2.mp4

.

### CODE REVIEW

**Review by sakleprajwal — APPROVED** [review]
Looks good. please go ahead.

File: src/Components/Product/Products.jsx [hunk 1]
```diff
@@ -1,80 +1,46 @@
...
+
+    useEffect(() => {
+
+        (async () => {
+            try {
+                const response = await fetch('/api/products', { method: "GET" });
+                const json = await response.json()
+                setProductsData(json.products)
+            } catch (err) {
+                console.log(err)
```
Comment (ranitasahaa1230) [inline src/Components/Product/Products.jsx:14]: Remove this console.log if not needed or else you can display on the UI - the error message

**Review by ranitasahaa1230 — APPROVED** [review]
Looks good to merge

File: src/Components/Product/Products.jsx (hunk 1 above)
Comment (Mahi24-prog) [reply src/Components/Product/Products.jsx:14]: ok

**Pull request merged**